src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,45 @@
-import React from 'react';
-
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  movie: Movie;
+};
```

Comment:
please choose more descriptive names for types 😉  maybe `MovieCardProps`?

Reply:
done!